src/seedsigner/views/seed_views.py

```diff
@@ -37,7 +37,6 @@ def __init__(self):
         for seed in self.controller.storage.seeds:
             self.seeds.append({
                 "fingerprint": seed.get_fingerprint(self.settings.get_value(SettingsConstants.SETTING__NETWORK)),
-                "has_passphrase": seed.passphrase is not None
```

Comment:
Not needed since no longer wanting to add lock icon for seeds with passphrases (see [here](https://github.com/SeedSigner/seedsigner/pull/286#issuecomment-1363351268).Removing unnecessary code.

Reply:
TY @EverydayBitcoiner, this code cleanup, less the trailing comma on the previous item, has been included in pr #324